vue/src/AdminPage/AdminPage.vue

```diff
@@ -110,10 +111,9 @@ export default defineComponent({
   },
   computed: {
     cliToolUsageText() {
-      const link = 'http://developer.matomo.org/guides/piwik-on-the-command-line';
       return translate(
         'VisitorGenerator_CliToolUsage',
-        `<a rel="noreferrer noopener" target="_blank" href="${link}">`,
+        externalLink('http://developer.matomo.org/guides/piwik-on-the-command-line'),
```

Comment:
Same here

Reply:
The parameters are being added in my tests:

![image](https://github.com/matomo-org/plugin-VisitorGenerator/assets/88810029/7a6ad51a-bec8-4d1d-abbf-8d1094393ba2)

Reply:
@bx80 It was cache its shows up now